mobile/src/main/java/com/gelakinetic/mtgfam/helpers/SearchCriteria.java

```diff
@@ -32,5 +32,5 @@
     public String collectorsNumber = null;
     public String colorIdentity = "wubrgl";
     public int colorIdentityLogic = 0;
-    public boolean noTokens;
+    public boolean moJhoStoFilter;
```

Comment:
Renamed the filter to make explicit that this filter is for MoJhoSto only. So no worries about future applications.